.github/workflows/ci_e2e-custom.yaml

```
@@ -25,6 +25,14 @@ jobs:
                 php: [ "8.1" ]
                 symfony: [ "^6.4" ]
                 mysql: [ "8.0" ]
+                resource: ["^1.10"]
+                grid: [ "^1.12" ]
+                include:
+                    -   php: "8.1"
+                        symfony: "^6.4"
+                        mysql: "8.0"
+                        resource: "^1.11@dev"
+                        grid: "^1.13@dev"                
```

Checking the last stable versions is not needed, as we checked that in other builds:
```suggestion
                resource: ["^1.11@dev"]
                grid: [ "^1.13@dev" ]
          
```